lib/departure/version.rb

```diff
@@ -1,3 +1,3 @@
 module Departure
-  VERSION = '6.3.0'.freeze
+  VERSION = '6.4.0'.freeze
```

Comment:
Would you mind reverting this change?

While I understand that this needs a version bump, I'm actually not sure if we need a minor or major bump on this if we have to drop support of some Ruby/Rails version due to compatibility.

Reply:
Fixed